editor/js/editable-wat.js

```diff
@@ -8,51 +8,48 @@ import "../css/editor-libs/ui-fonts.css";
 import "../css/editor-libs/common.css";
 import "../css/editable-js-and-wat.css";
 import "../css/editor-libs/tabby.css";
-import {getEditorContent, initCodeEditor, languageJavaScript, languages, languageWAST} from "./editor-libs/codemirror-editor.js";
-
-(function () {
-  var watCodeBlock = document.getElementById("static-wat");
-  var jsCodeBlock = document.getElementById("static-js");
-  var exampleFeature = watCodeBlock.dataset["feature"];
-  var execute = document.getElementById("execute");
-  var liveContainer = "";
-  var output = document.querySelector("#console code");
-  var reset = document.getElementById("reset");
-  var wabtInitialized = wabtConstructor();
-
-  var tabContainer = document.getElementById("tab-container");
-  var tabs = tabContainer.querySelectorAll("button[role='tab']");
-  var tabList = document.getElementById("tablist");
-
-  var watCodeMirror;
-  var jsCodeMirror;
-  var staticContainer;
-  var wabt;
-  wabtInitialized.then((res) => {
-    wabt = res;
-  });
+import {getEditorContent, initCodeEditor, languageJavaScript, languageWAST} from "./editor-libs/codemirror-editor.js";
+
+(async function () {
+  const watCodeBlock = document.getElementById("static-wat");
+  const jsCodeBlock = document.getElementById("static-js");
+  const exampleFeature = watCodeBlock.dataset["feature"];
+  const execute = document.getElementById("execute");
+  const output = document.querySelector("#console code");
+  const reset = document.getElementById("reset");
+  const wabtInitialized = wabtConstructor();
```

Comment:
Now that we're using ESM and defining an async function, we no longer need a separate variable to handle the initialization of `wabt`.

```suggestion
  const wabt = await wabtConstructor();
```